spec/integration/system_pager_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+RSpec.describe TTY::Pager::SystemPager do
+  around :each do |example|
+    Dir.mktmpdir do |dir|
+      FileUtils.cd(dir) do
+        example.run
+      end
+    end
+  end
+
+  it "paginates some text" do
+    output_path = Pathname.new('output.txt')
+
+    described_class.page("Some text", command: "cat > #{output_path}")
```

Comment:
I wonder if it wouldn't be better to make the `cat` like a utility in Ruby to ensure it's system independent. For example:

```ruby
File.open("cat.rb", "w") do |script|
  script.write <<-EOS
    puts ARGF.read
  EOS
end
```

Reply:
Yes, this makes sense. Originally, I was planning on both pagers to be different, one of them stopping execution at the second line. But I admit it would be nicer to have them out of the actual test.